hoodie-client/src/main/java/com/uber/hoodie/config/HoodieWriteConfig.java

```diff
@@ -338,6 +342,39 @@ public String getGraphiteMetricPrefix() {
     return new Builder();
   }
 
+  /**
+   * memory configs
+   */
+  public double getMaxMemoryFractionPerPartitionMerge() {
+    return Double.valueOf(props.getProperty(HoodieMemoryConfig.MAX_MEMORY_FRACTION_FOR_MERGE_PROP));
+  }
+
+  public static String getSparkExecutorMemoryProp() {
+    return HoodieMemoryConfig.SPARK_EXECUTOR_MEMORY_PROP;
+  }
+
+  public static String getSparkExecutorMemoryFractionProp() {
+    return HoodieMemoryConfig.SPARK_EXECUTOR_MEMORY_FRACTION_PROP;
+  }
+
+  public static String getDefaultMaxMemoryInBytesForMerge() {
+    return HoodieMemoryConfig.DEFAULT_MAX_MEMORY_IN_BYTES_FOR_MERGE;
+  }
+
+  public static String getDefaultSparkExecutorMemoryFraction() {
+    return HoodieMemoryConfig.DEFAULT_SPARK_EXECUTOR_MEMORY_FRACTION;
+  }
+
+  public static String getDefaultSparkExecutorMemory() {
```

Comment:
why explicit getters for already public static constants?  That said, I recommend we localize all this to the config class

Reply:
So the way our HoodieConfigs are setup, we expose getters from a single HoodieWriteConfig and then depend on the constants from the specific configs, hence those constants need to be public so they can be accesses in HoodieWriteConfig and I want one way for client to access them (via getters) rather than using the public constants, this isn't new, the same applied already for compaction, metrics...